.github/workflows/alpha.yaml

```diff
@@ -0,0 +1,406 @@
+        key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-mod }}
+        key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+
+    - name: Get tags
+      id: get_tag
+      uses: ./actions/version-tag
+
+    - name: Download web artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: web
+        path: ./web/dist
+
+    - name: Build Go API
+      env:
+        GIT_COMMIT: ${{ github.sha }}
+        # GITHUB_SHA: ${{ github.sha }}
+        GIT_TAG: ${{ steps.get_tag.outputs.GIT_TAG }}
+        # GITHUB_REPOSITORY:
+        # GITHUB_WORKSPACE:
+        SCOPE_DSN_PUBLIC: ""
+      run: make test kotsadm
+
+    - name: Upload Go API artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: go_api
+        path: ./bin
+
+
+  release_go_api_alpha:
+    runs-on: ubuntu-18.04
+    needs: [build_web, build_go_api]
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.3'
+
+    - id: go-cache-paths
+      run: |
+        echo "::set-output name=go-build::$(go env GOCACHE)"
+        echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-build }}
+        key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-mod }}
+        key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+
+    - name: Download go_api artifact
+      uses: actions/download-artifact@v2
+      with:
+        name: go_api
+        path: ./bin
+
+    - name: Add executable permissions
+      run: chmod a+x ./bin/kotsadm
+
+    - uses: azure/docker-login@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+    - name: Build alpha release
+      env:
+        GIT_COMMIT: ${{ github.sha }}
+      run: |
+        make build-alpha
+
+
+  release_go_api_tagged:
+    runs-on: ubuntu-18.04
+    needs: [build_web, build_go_api]
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.3'
+
+    - id: go-cache-paths
+      run: |
+        echo "::set-output name=go-build::$(go env GOCACHE)"
+        echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-build }}
+        key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
+    - uses: actions/cache@v2
+      with:
+        path: ${{ steps.go-cache-paths.outputs.go-mod }}
+        key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
+
+    - name: Get tags
+      id: get_tag
```

Comment:
this isn't used.  (i know you didn't add it)

Reply:
is this gonna be an empty string in this entire workflow? should i remove all references?

Reply:
yep